src/app_service/service/connector/service.nim

```diff
@@ -0,0 +1,63 @@
+import NimQml, chronicles, json
+
+import backend/connector as status_go
+
+import app/global/global_singleton
+
+import app/core/eventemitter
+import app/core/signals/types
+
+import strutils
+
+logScope:
+  topics = "connector-service"
+
+const SIGNAL_CONNECTOR_SEND_REQUEST_ACCOUNTS* = "ConnectorSendRequestAccounts"
+
+# Enum with events
+type Event* = enum
+  DappConnect
+
+# Event handler function
+type EventHandlerFn* = proc(event: Event, payload: string)
+
+# This can be ditched for now and process everything in the controller;
+# However, it would be good to have the DB based calls async and this might be needed
+QtObject:
+  type Service* = ref object of QObject
+    events: EventEmitter
+    eventHandler: EventHandlerFn
+
+  proc delete*(self: Service) =
+    self.QObject.delete
+
+  proc newService*(
+    events: EventEmitter
+  ): Service =
+    new(result, delete)
+    result.QObject.setup
+
+    result.events = events
+
+  proc init*(self: Service) =
+    self.events.on(SignalType.ConnectorSendRequestAccounts.event, proc(e: Args) =
+      if self.eventHandler == nil:
+        return
+
+      var data = ConnectorSendRequestAccountsSignal(e)
+
+      if not data.requestID.len() == 0:
+        echo "ConnectorSendRequestAccountsSignal failed, requestID is empty"
```

Comment:
leftover, or use `error` from `chronicles`

Reply:
+1 to using `error`